pages/index.html

```diff
@@ -60,44 +54,43 @@ <H1>{% trans "DJ Your Way. For Free." %}</H1>
-  <p style="text-align: center;">The most advanced free DJ software can be yours in seconds.</p>
+  <p style="text-align: center;">Free and open source DJ software for Windows, macOS, and Linux</p>
   <div style="clear: both;"></div>
 </div> <!-- gapfiller -->
 
 <div class="content">
-  <H1>{% trans "Incredible Features, Unbeatable Value" %}</H1>
+  <h1>{% trans "Powerful Features For All DJs" %}</h1>
 
-  {% blocktrans %}Mixxx has everything you need to start making DJ mixes in a tight, integrated package. Whether you're DJing your next house party, spinning at a club, or broadcasting as a radio DJ, Mixxx has what you need to do it right.{% endblocktrans %}
+  {% blocktrans %}Mixxx integrates the tools DJs need to perform creative live mixes with digital music files. Whether you are a new DJ with just a laptop or an experienced turntablist, Mixxx can support your style and techniques of mixing.{% endblocktrans %}
 
   <div class="halfbox_left">
-    <H2><img src="{% static '/static/images/ic_library_itunes.png' %}" class="feature_icon">{% trans "iTunes Integration" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_keydetect.png' %}" class="feature_icon">{% trans "BPM, Key Detection and Sync" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}All your playlists and songs from iTunes, automatically ready to go for your next live DJ performance.{% endblocktrans %}
+      {% blocktrans %}BPM and musical key detection help you find the perfect next track from your library. Use master sync to match the tempo and beats of four songs for seamless mixing.{% endblocktrans %}
     </p>
   </div>
 
   <div class="halfbox_right">
-    <H2><img src="{% static '/static/images/ic_preferences_controllers.png' %}" class="feature_icon">{% trans "DJ Controller Support" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_controllers.png' %}" class="feature_icon">{% trans "DJ Controller Support" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}With over 85 MIDI DJ controllers supported out-of-the-box and several HID controllers, Mixxx gives you comprehensive hardware control for your DJ mixes.{% endblocktrans %}
+      {% blocktrans %}Built-in mappings for DJ controller hardware gives you hands-on control of Mixxx's features. Use the programmable mapping system to customize your workflow and add support for any MIDI or HID device.{% endblocktrans %}
     </p>
   </div>
 
   <div style="clear: both;"></div>
 
   <div class="halfbox_left">
-    <H2><img src="{% static '/static/images/ic_library_prepare.png' %}" class="feature_icon">{% trans "BPM Detection and Master Sync" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_waveform.png' %}" class="feature_icon">{% trans "Effects" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}Instantly sync the tempo of four songs for seamless beatmixing. Need a break? Create a quick playlist and let Auto DJ take over.{% endblocktrans %}
```

Comment:
autodj is a very popular feature with our users, we shouldn't drop it.

Reply:
Eh, I don't see an easy way to fit it into the 2 x 2 grid of highlighted core features.